frontend/projects/valtimo/case/src/lib/components/case-detail/tab/widgets/components/external-plugin/case-widget-external-plugin.component.scss

```
@@ -0,0 +1,100 @@
+    color: var(--widget-text-color, var(--cds-text-primary));
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  &__body {
+    display: flex;
+    flex-direction: column;
+    flex: 1 1 auto;
+    margin-top: 16px;
+    // The iframe has no intrinsic height (no host-side resize handling), so the plugin canvas gets a
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)